Mathlib/Data/FunLike/Basic.lean

```diff
@@ -57,15 +59,15 @@ Continuing the example above:
 ```
 /-- `MyHomClass F A B` states that `F` is a type of `MyClass.op`-preserving morphisms.
 You should extend this class when you extend `MyHom`. -/
-class MyHomClass (F : Type*) (A B : outParam <| Type*) [MyClass A] [MyClass B]
-  extends DFunLike F A (λ _, B) :=
+class MyHomClass (F: Type*) (A B : outParam Type*) [MyClass A] [MyClass B]
```

Comment:
This would be a great place to explain when one should use outParams, and when not. Also, you could explain the reason of the unbundled design here, with a reference to the PR.

Also,
```suggestion
class MyHomClass (F : Type*) (A B : outParam Type*) [MyClass A] [MyClass B]
```